Shrine/app/src/main/java/com/authentication/shrine/ui/viewmodel/RegistrationViewModel.kt

```diff
@@ -111,7 +106,7 @@ class RegistrationViewModel @Inject constructor(
     fun createRestoreKey(
         createRestoreKeyOnCredMan: suspend (createRestoreCredRequestObj: JSONObject) -> GenericCredentialManagerResponse,
     ) {
-        viewModelScope.launch {
+        GlobalScope.launch {
```

Comment:
why do we need global scope in viewmodel as it is going to be destroyed once work done and we move to next screen, can you check - it might cause a leak.

Reply:
Open for discussion on this one. In this coroutine, we are launching a new screen. The new screen gets launched before the call is completed. And as soon as the new screen is launched this viewModelScope gets cancelled. So for now I have added GlobalScope.